Add `device_type` in metadata (#21347)
DataDog/documentation: content/en/network_monitoring/devices/profiles.md

For more Datadog provided profiles, see the [GitHub repository][3].

### Metadata definition by profile

> Profiles have a metadata section that can be used to define where and how metadata should be collected. Values can be static or come from an OID value.

Reviewer: ```suggestion
In the profiles's metadata section, you can define where and how metadata is collected. Values can be static or come from an OID value.
```

Author: addressed in [87b8c3e](https://github.com/DataDog/documentation/pull/21347/commits/87b8c3e330917fd1b1ac4d3e80c63b47471e152f)

> Supported fields can be found [here][4].

Reviewer: Is there a specific section (to replace `this section`) in the code that we can point users to?

```suggestion
See [this section][4] for the supported fields.
```

Author: would `See the [DeviceMetadata][4] for the supported fields` work ?
and I'll update the link to point to the section instead of the whole file

Reviewer: Thanks! I updated the suggestion.

Author: addressed in [0bc14c3](https://github.com/DataDog/documentation/pull/21347/commits/0bc14c33ec658b543030acd309dd209296be0569)

> With Datadog Agent version 7.52 and later, a new `device_type` field is supported as a device metadata. This can be set manually in the profile and can be used to filter on specific types of devices. Accepted values include:

Reviewer: ```suggestion
With Datadog Agent version 7.52 and later, there is a `device_type` field for device metadata. This can be set manually in the profile and can be used to filter on specific types of devices. Accepted values include:
```

- access_point
- firewall
- load_balancer
- pdu
- printer
- router
- sd-wan
- sensor
- server
- storage
- switch
- ups
- wlc

> To know more about the profiles format, see the [Profile Format Reference][5].

Reviewer: ```suggestion
See the [Profile Format Reference][5] for more information about profiles formats.
```

Author: adresed in [5c48a7a](https://github.com/DataDog/documentation/pull/21347/commits/5c48a7a914c0a2cc53f4c4f2498b140f9fa8c7ef)
